### PR TITLE
Add CoM shifter to proc avionics

### PR DESCRIPTION
**Pull request by nepphhh**

### CODE REVIEW

**Comment by siimav** [comment]
Do you ever even look at the changes you commit? Please make it a habit to double and triple check all your changes before pushing them into a PR. This PR is full junk files.

File: GameData/RP-0/Tree/ProceduralAvionics.cfg [hunk 1]
```diff
@@ -27,6 +27,15 @@
 		}
 	}
 	
+	MODULE {
```
Comment (siimav) [inline GameData/RP-0/Tree/ProceduralAvionics.cfg]: Once again you're not following the same style as the other configs in this file.

File: GameData/RP-0/Tree/ProceduralAvionics.cfg [hunk 2]
```diff
@@ -27,6 +27,15 @@
...
+		name = AdjustableCoMShifter
+		offsetX = 0
+		offsetY = 0
+		offsetZ = 0
+		offsetPercent = 0.2
+		IsDescentMode = false
```
Comment (siimav) [inline GameData/RP-0/Tree/ProceduralAvionics.cfg]: No need to define default values inside the patch. Also offsetPercent and IsDescentMode are only used on capsules for the Descent mode functionality.

File: GameData/RP-0/Tree/ProceduralAvionics.cfg (hunk 1 above)
Comment (nepphhh) [reply GameData/RP-0/Tree/ProceduralAvionics.cfg]: Old habits die hard

File: GameData/RP-0/Tree/ProceduralAvionics.cfg (hunk 2 above)
Comment (nepphhh) [reply GameData/RP-0/Tree/ProceduralAvionics.cfg]: Thanks, I was unaware of this from asking people & reading the documentation. Well, not really any documentation to read. 

So what changes should I make? Remove the offsetPercent and IsDescentMod nodes?

Comment (siimav) [reply GameData/RP-0/Tree/ProceduralAvionics.cfg]: Yes, remove all the lines that I marked above.

**Comment by nepphhh** [comment]
Changes made.

**Pull request merged**